docs/rfcs/0013-rfc-process-updates.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+title: RFC process updates
+permalink: /docs/rfcs/0013-rfc-process-updates
+---
+
+* RFC PR: https://github.com/mozilla-mobile/firefox-android/pull/5681
+* Start date: 2024-02-20 
+* End date: 2024-03-22 
+* Stakeholders: @jonalmeida, @boek
+
+## Summary
+
+Clarify requirements for RFCs by introducing a README and a template.
+
+## Motivation
+
+- The scale of the Firefox Android team has grown substantially since the RFC process was first introduced. 
+- More external teams are using and contributing to Android Components. 
+- Approval or rejection of a proposal has required periods of time that can slow implementation.
+- Stakeholders have either been implicit by sharing RFCs in various communication channels, or an author must be expected to correctly select reviewers that lack explicitly documented ownership of affected areas.
```

Comment:
It's fine to have secondary channels of communication for feedback - everyone has their own way. The missing step that I've seen in practice, is that developers forget to come back to the origin (their original doc or bug or PR), and document the communication and it's outcome.

If we want specific feedback from a person, we add them as a reviewer to the PR today. It's up to the author to decide then if they want to wait for that feedback or they are happy with the review of another reviewers.

Below it says, "explicit stakeholders for RFCs" which contradicts, "an author must be expected to correctly select reviewers.."

I'd recommend instead making this into something similar to: "New RFCs may not be visible to the entire team to have equal time to consider."

The proposal could then be: "Share the RFC to the mailing list/slack channel with the deadline date."

For prior art of what I'm suggesting, see the web standards process or the `services-rfcs@` mailing list.

Reply:
> It's fine to have secondary channels of communication for feedback - everyone has their own way.

I am hoping this removes some of the guesswork in finding out what every reviewer's or author's individual way is, especially for people that are new to the process or working across teams/areas of ownership.

> Below it says, "explicit stakeholders for RFCs" which contradicts, "an author must be expected to correctly select reviewers.."

In the way that selecting explicit stakeholders and reviewers is basically the same thing? I do agree here, but I am hoping with a revised CODEOWNERS file that this is improved and that by offering the suggestion as part of the guide and template, it will give authors an explicit step in seeking feedback.

> The proposal could then be: "Share the RFC to the mailing list/slack channel with the deadline date."
> For prior art of what I'm suggesting, see the web standards process or the services-rfcs@ mailing list.

The services-rfcs mailing list looks great. I would definitely like to include something like that as part of the guide for distribution of these, but I don't feel like the firefox-android mailing list is all-inclusive enough especially as we talk about other teams potentially taking ownership of specific areas. Do you have any suggestions, or should we make a new one?

Reply:
> My impression is that we are solving for the wrong problem then. 🤷

Is there another problem you think we should be solving for? I am basing this proposal off my own and other's feedback that a lack of clarity has lowered engagement and speed of this process, but I am open to try and solve additional problems with this.